config/rbac/role.yaml

```diff
@@ -70,5 +70,6 @@ rules:
   verbs:
   - get
   - list
+  - patch
+  - update
```

Comment:
because I am adding and removing finalizer to the config map, so I need to update config map

I can remove patch, I just thought it is similar to update so I've added that as well